cnf-certification-test/operator/suite.go

```diff
@@ -70,17 +71,20 @@ func testOperatorInstallationPhaseSucceeded(env *provider.TestEnvironment) {
 		// could be restarting. Let's give it some time before declaring it failed.
 		phase := phasecheck.WaitOperatorReady(csv)
 		if phase != v1alpha1.CSVPhaseSucceeded {
-			badOperators = append(badOperators, env.Operators[i].String())
+			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewOperatorReportObject(env.Operators[i].Namespace, env.Operators[i].Name, "Operator not in Succeeded state ", false))
```

Comment:
Add a new field for the operator named phase or operatorPhase and add it to the operator object with addField